sdk/storage/Azure.Storage.Files.DataLake/api/Azure.Storage.Files.DataLake.netstandard2.0.cs

```diff
@@ -933,6 +933,7 @@ public static partial class PathAccessControlExtensions
         public static string ToAccessControlListString(System.Collections.Generic.IList<Azure.Storage.Files.DataLake.Models.PathAccessControlItem> accessControlList) { throw null; }
         public static string ToOctalRolePermissions(this Azure.Storage.Files.DataLake.Models.RolePermissions rolePermissions) { throw null; }
         public static string ToSymbolicRolePermissions(this Azure.Storage.Files.DataLake.Models.RolePermissions rolePermissions) { throw null; }
+        public static string ToSymbolicRolePermissions(this Azure.Storage.Files.DataLake.Models.RolePermissions rolePermissions, bool stickyBit = false) { throw null; }
```

Comment:
Can we drop the `= false` here?  Ideally `= false` would be what you get when you use the other existing overload.  It's pretty confusing for most customers which overload gets invoked when they pass `ToSymbolicRolePermissions(permissions)` and trawling through https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/expressions#1164-overload-resolution is no fun.

Reply:
Done